doc/tools/make_rst.py

```diff
@@ -941,6 +941,33 @@ def escape_rst(text, until_pos=-1):  # type: (str) -> str
     return text
 
 
+def generate_rst_filename(class_def, state):  # type (ClassDef, State) -> str
+    file_prefix = "class"
+    class_name = class_def.name
+
+    if class_name.startswith("@"):
+        file_prefix = "global"
+    else:
+        if class_def.inherits:
```

Comment:
nit: how about merging `else:` from the line above together with this `if` into `elif class_def.inherits:`. It would save one level of indentation which would look nicer.